.github/workflows/ci.yml

```diff
@@ -192,4 +192,38 @@ jobs:
           source cacitesting.env
           python -m unittest discover -s tests/skr
 
+  test-encfs:
+    name: Test EncFS
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+
+      - name: Install c_aci_testing package
+        env:
+          GH_TOKEN: ${{ github.token }}
+        run: ./scripts/install-c-aci-testing.sh
+
+      - name: Setup Docker
+        run: sudo usermod -aG docker $USER
+
+      - name: Log into Azure
+        uses: azure/login@v2
+        with:
+          client-id: ${{ secrets.MANAGED_ID_CLIENT_ID }}
+          tenant-id: ${{ secrets.MANAGED_ID_TENANT_ID }}
+          subscription-id: ${{ vars.SUBSCRIPTION }}
+
+      - name: Test EncFs
```

Comment:
Nit: consistency with the capitalization of ENCFS. I usually do all caps to match SKR despite the capitalization not matching the number of words.

Reply:
Done in 6848b91e252ea9fa4afc4bff0f529d380b72ff7c